beets/dbcore/types.py

```diff
@@ -168,52 +209,66 @@ def format(self, value: int) -> str:
         return '{}{}'.format((value or 0) // self.unit, self.suffix)
 
 
-class Id(Integer):
+class Id(NullInteger):
```

Comment:
I understand why this is slightly worrisome, but it is clearly the right thing to do and it's hard to imagine how this could actually break anything…